python/kserve/kserve/protocol/dataplane.py

```diff
@@ -23,6 +23,7 @@
 from ray.serve.handle import RayServeHandle, RayServeSyncHandle, DeploymentHandle
 
 from .rest.v2_datamodels import GenerateRequest, GenerateResponse
+from .rest.openai_datamodels import ChatCompletionRequest, ChatCompletionResponse
```

Comment:
Also could the types from openai pkg be used here?

Reply:
Late response but yes, we're also using the types directly from `openai-python` in this PR.